.github/workflows/test-internal-prs.yml

```diff
@@ -0,0 +1,169 @@
+# Description: This workflow runs test suite against PRs that are not from forks.
+#
+# Triggered by:
+#   (1) Opening a pull request
+#   (2) On every commit to the PR
+#   (3) Adding run-tests label to the PR
```

Comment:
The reason why we still have `run-tests` label on internal PR is because workflows will not run against bot-generated PRs (e.g. "Version Packages" PR!) unless there's an human action. Adding a label counts as a human action.

That's actually why we add `run-tests` label on version packages, even though that PR is not from fork.